ament_tools/verbs/test/cli.py

```diff
@@ -46,6 +46,10 @@ def prepare_arguments(parser, args):
         default=False,
         help='Stop after tests with errors or failures',
     )
+    parser.add_argument(
+        '-n', '--no-build', dest='build_before_test', default=True, action='store_false',
+        help="assumes package is built, only running the tests"
```

Comment:
To match other arguments I would suggest:

    help='Skip building package, assuming it has been built before'